packages/react/src/__helpers__/setupTestScenario.ts

```diff
@@ -0,0 +1,23 @@
+import { MockedResponse } from '@apollo/client/testing';
+import { mockLensApolloClient } from '@lens-protocol/api-bindings/mocks';
+import { RenderHookResult } from '@testing-library/react';
+
+import { defaultMediaTransformsConfig } from '../mediaTransforms';
+import { renderHookWithMocks } from './testing-library';
+
+export function setupTestScenario(mocks: MockedResponse[]) {
```

Comment:
nit: this exact wording (`setupTestScenario`) is used in many places with very local meaning...especially with things not associate with React, not sure if this is intelligible enough... I would consider:

- `setupUITestScenario`
- `setupReactTestScenario`
- `setupHookTestScenario`